packages/config/config/devices/0x015f/mh-dt311_411.json

```diff
@@ -168,9 +173,10 @@
 			"allowManualEntry": true
 		},
 		"9": {
-			"label": "Single Dimming Step Time (Auto)",
-			"unit": "10 ms",
+			"label": "Dimming Step Time (Auto)",
```

Comment:
Does the manual say Auto here, or Z-Wave? Typically the dim steps are defined for manual, z-wave, and all on/off.

Reply:
Yes the manual does mention "Auto" - seems that the dim steps correspond to three situations:

![image](https://user-images.githubusercontent.com/34757455/115202187-bad1af80-a128-11eb-9e9f-011262eb9eab.png)